argocd/ctrl_plane/dev/kubernetes-secret-generator.yaml

```
@@ -0,0 +1,25 @@
+metadata:
+  name: kubernetes-secret-generator
+  namespace: argocd
+spec:
+  project: default
+  source:
+    chart: kubernetes-secret-generator
+    repoURL: https://helm.mittwald.de
+    targetRevision: 3.4.0
+    helm:
+      valuesObject:
+          nodeSelector:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)